experimenter/experimenter/nimbus-ui/src/components/PageSummary/index.test.tsx

```diff
@@ -144,6 +144,27 @@ describe("PageSummary", () => {
     });
   });
 
+  it("can cancel review when dirty rollout is in the review state", async () => {
+    const { mockRollout, rollout } = mockLiveRolloutQuery("demo-slug", {
+      ...reviewRequestedBaseProps,
+      canReview: true,
+      isRolloutDirty: true,
+    });
+    const mutationMock = createFullStatusMutationMock(
+      rollout.id!,
+      NimbusExperimentStatusEnum.LIVE,
+      NimbusExperimentStatusEnum.LIVE,
+      NimbusExperimentPublishStatusEnum.REVIEW,
+      CHANGELOG_MESSAGES.CANCEL_REVIEW,
+    );
+    render(<Subject mocks={[mockRollout, mutationMock]} />);
+
+    await screen.findByTestId("cancel-review-start");
+    fireEvent.click(screen.getByTestId("cancel-review-start"));
+
+    screen.queryByTestId("pill-dirty-unpublished");
```

Comment:
with this you can also say both pill and request update button is present after cancellation `screen.queryByTestId("request-update-button");`

Reply:
good idea!

Reply:
:saluting_face: